libpod/runtime.go

```diff
@@ -143,7 +143,7 @@ type RuntimeConfig struct {
 	// to attach pods to
 	CNIDefaultNetwork string `toml:"cni_default_network,omitempty"`
 	// HooksDir Path to the directory containing hooks configuration files
-	HooksDir string `toml:"hooks_dir"`
+	HooksDir []string `toml:"hooks_dir"`
```

Comment:
Does this break compatability in libpod.conf?

Reply:
It looks like yes, but we weren't including it in the default at least

Reply:
Right know one would know about it.  I actually don't think we should save it there anyways.  This is not documented and is only for testing.